spec/serializers/comment_serializer_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "spec_helper"
+
+describe Export::CommentSerializer do
+  let(:comment) { create(:comment) }
+  subject(:json_output) { Export::CommentSerializer.new(comment).to_json }
+
+  it { is_expected.to include("\"guid\":\"#{comment.guid}\"") }
```

Comment:
Actually it would be good to use percent literals for all these to avoid all the escaping:

```ruby
it { is_expected.to include %("guid":"#{comment.guid}") }
```